tests/output.rs

```diff
@@ -150,6 +165,14 @@ fn massage_error_output(output: &str) -> String {
     output.replace(".exe", "")
 }
 
+fn build_example(example_name: &str) {
+    // CargoBuild::new().example(example_name).exec().unwrap();
```

Comment:
This was weird, per [this answer](https://stackoverflow.com/a/75564431/732366) I tried using [`escargot`](https://docs.rs/escargot/latest/escargot/index.html) to "properly" tell it to build a certain example but this seemed to be hanging when I ran the tests

So fell back to just manually creating a `cargo` command line? The one thing I notice there is that you see some "Blocking waiting for file lock on build directory"'s in the `cargo test` output but doesn't seem to actively cause any problems